extensions/default/src/Panels/requestDisplaySetCreationForStudy.js

```diff
@@ -13,7 +13,7 @@ function requestDisplaySetCreationForStudy(
     return;
   }
 
-  dataSource.retrieve.series.metadata({ StudyInstanceUID, madeInClient });
+  return dataSource.retrieve.series.metadata({ StudyInstanceUID, madeInClient });
```

Comment:
There seems to be two files named `requestDisplaySetCreationForStudy` , can we re-use one